_rules/label-content-name-mismatch-2ee8b8.md

```diff
@@ -93,6 +93,14 @@ Not all of visible label is included in accessible name.
 <button name="link" aria-label="the full">The full label</button>
 ```
 
+#### Failed Example 3
+
+Mathematical symbols can not be substitute for text.
+
+```html
+<a href="/" aria-label="Page two">Page 2</a>
```

Comment:
can we add examples with other mathematical operators (+,-,*,/).

Reply:
In the spirit of not overloading the test cases, I'd prefer not to. Anyone else things this is essential? @kasperisager @carlosapaduarte @annethyme?

Reply:
I do think calling numerals "mathematical symbols" is a bit of a stretch.

Reply:
Indeed, I have to agree with @kasperisager 
That would be appropriate if the example included a mathematical formula, but here the issue is not related with the interpretation of mathematical symbols, but with potential translation issues.
My suggestion would be to retain this example but with a different description, and add an example with a mathematical formula with this example's description.

Reply:
Can we have one example that includes both?

`<a href="/" aria-label="Proof of one plus one is two">Proof of 1+1=2</a>`